web/web_test.go

```diff
@@ -15,6 +15,20 @@ import (
 
 // TestDownloadFile tests the behavior of the DownloadFile
 // function in different scenarios
+type ErrorAssertionFunc func(t *testing.T, err error)
+
+func NoError(t *testing.T, err error) {
+    if err != nil {
+        t.Fatalf("expected no error but got: %v", err)
+    }
+}
+
+func Error(t *testing.T, err error) {
+    if err == nil {
+        t.Fatalf("expected an error but got none")
+    }
+}
+
```

Comment:
You don't really need these helper funcs you can just use `require.NoError` and `require.Error` respectively.

Reply:
@Ajpantuso OK - I see how this works now.